src-tauri/ai2html-config.json

```diff
@@ -1,7 +1,6 @@
 {
   "html_output_path": "./ai2html-output/",
   "html_output_extension": ".html",
-  "include_resizer_script": false,
```

Comment:
We did, just because we handle which artboard to display with our new CSS. This is how it appears in this repo:  https://github.com/The-Politico/test_ai2html-2022, which was our latest review of ai2html for Politico.